obspy/io/rg16/__init__.py

```diff
@@ -4,8 +4,8 @@
 
 Functions to read waveform data from Receiver Gather 1.6-1 format.
 This format is used to store continuous data recorded by
-`Farfield Nodal <fairfieldnodal.com>`_'s
-`Zland <http://fairfieldnodal.com/equipment/zland>`_ product line.
+``Farfield Nodal <fairfieldnodal.com>``_'s
+``Zland <http://fairfieldnodal.com/equipment/zland>``_ product line.
```

Comment:
I'm sorry if I confused you earlier @thefroid.. these links should stay as before. Please see the autogenerated PR docs here: http://docs.obspy.org/pull-requests/2265/packages/obspy.io.rg16.html?highlight=fairfield

Also see this page for reference: http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html